source/_integrations/verisure.markdown

```diff
@@ -38,6 +38,10 @@ There is currently support for the following device types within Home Assistant:
 
 {% include integrations/config_flow.md %}
 
+## Configuration options
+
+- Number of digits in PIN code for locks
```

Comment:
This is actually part of the options flow. We have templates for that (which can be included).